revolv/project/models.py

```diff
@@ -333,6 +333,16 @@ def partial_completeness(self):
         ratio = self.amount_donated / float(self.funding_goal)
         return min(ratio, 1.0)
 
+    @property
+    def percent_complete(self):
+        """
+        :return: a float between 0 and 1, representing the completeness of this
+            project with respect to its goal (1 if exactly the goal amount, or
+            more, has been donated, 0 if nothing has been donated).
+        """
+        ratio = self.amount_donated / float(self.funding_goal)
```

Comment:
You can get rid of this line if you so `return int(self.partial_completeness() * 100)` in the next line